xain_fl/coordinator/coordinator.py

```diff
@@ -367,7 +367,7 @@ def _handle_end_training_round(
             self.weights = self.aggregator.aggregate(self.round.get_weight_updates())
 
             # update the round or finish the training session
-            if self.current_round == self.num_rounds:
+            if self.current_round >= self.num_rounds - 1:
```

Comment:
Isn't this equivalent to
```python
if self.current_round > self.num_rounds:
```
?

Reply:
no, but it is equivalent to
```python
if self.current_round > self.num_rounds - 2:
```

Reply:
oh :man_facepalming: I read this as `+ 1` instead of `- 1`, sorry.

Reply:
no, it is equivalent to `self.current_round > self.num_rounds - 2`.